modules/cart/commerce_cart.module

```diff
@@ -191,6 +192,26 @@ function commerce_cart_form_commerce_order_type_form_alter(array &$form, FormSta
     '#default_value' => $cart_block_view,
   ];
 
+  $form['commerce_cart']['enable_expiration'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Enable cart expiration for this order type'),
+    '#default_value' => !empty($cart_expiration),
+  ];
+
+  $form['commerce_cart']['cart_expiration'] = [
+    '#type' => 'number',
+    '#title' => t('Days until cart expires'),
+    '#description' => t('The cart will be removed in the number of days specified. Set to 0 disable cart expiration.'),
```

Comment:
Typo here, it should be: 

'#description' => t('The cart will be removed in the number of days specified. Set to 0 **to** disable cart expiration.'),